docs/governance/community-guidelines/templates/features.yaml

```diff
@@ -0,0 +1,10 @@
+title: CCC <Service Category> Security Threats
```

Comment:
Jusst checking that this should have the title "Security Threats" and it wasn't a copy paste?